src/TestSuite/IntegrationTestCase.php

```diff
@@ -343,11 +367,11 @@ protected function _buildRequest($url, $method, $data)
         ];
         $session = Session::create($sessionConfig);
         $session->write($this->_session);
-
         list ($url, $query) = $this->_url($url);
+
         $props = [
             'url' => $url,
-            'post' => $data,
+            'post' => $this->_addTokens($url, $method, $data),
```

Comment:
Sounds like a good idea, I came to the same conclusion last night. The consistency is more important than the automatic behavior.